chain/src/main/scala/org/bitcoins/chain/models/BlockHeaderDAO.scala

```diff
@@ -194,4 +194,16 @@ case class BlockHeaderDAO()(
       Future.sequence(nestedFuture)
     }
   }
+
+  /** Finds a [[BlockHeaderDb block header]] that satisfies the given predicate, else returns None */
```

Comment:
I don't think the link here will work in the Scaladoc, you'd need to do `[[org.bitcoins.chain.models.BlockHeaderDb block header]]`